src/static/css/main.less

```diff
@@ -52,7 +52,7 @@ nav ul {
 
 // custom modules
 @import (less) "header.less";
-@import (less) "hero.less";
+//@import (less) "hero.less";
```

Comment:
Should this just be deleted? Or fixed so that the hero on the sandbox homepage works again?